types/scripts/build-types.ts

```diff
@@ -128,7 +135,7 @@ function spawnWorkerd(
 async function buildEntrypoint(
   entrypoint: (typeof ENTRYPOINTS)[number],
   workerUrl: URL
-): Promise<void> {
+): Promise<{ name: string; files: Array<{ fileName: string; content: string }> }> {
```

Comment:
I'd suggest giving `Array<{ fileName: string; content: string }>` a type name, because it appears at least three times.

Reply:
Updated the next usage. This no longer applies.